falcon_multiqc/commands/save.py

```diff
@@ -148,8 +181,33 @@ def cli(directory, sample_metadata, batch_description, cohort_description, batch
             for line in batch_metadata:
                 split = line.split(",")
                 batch_name = split[0]
+                number_of_samples = split[1] # to be implmented (have to modify models)
                 batch_description = split[2]
                 # Update each batch with the given batch description.
-                (session.query(Batch)
-                    .filter(Batch.batch_name == batch_name, Batch.cohort_id == cohort_id)
-                    .one().description) = batch_description
+                try:
+                    (session.query(Batch).filter(Batch.batch_name == batch_name).one().description) = batch_description
+                    session.commit()
+                    click.echo(f"Batch '{batch_name}' description has been saved.")
+                except NoResultFound:
+                    click.echo(f"Batch '{batch_name}' is not present in the database so description cannot be added, exiting.")
+                    sys.exit(1)
+
+        # Save Cohort metadata.
+        if cohort_metadata:
+            # Skip header
+            next(cohort_metadata)
+            for line in cohort_metadata:
+                split = line.split(",")
+                cohort_name = split[0] # to be implmented (have to modify models)
+                number_of_samples = split[1] # to be implmented (have to modify models)
```

Comment:
why?

Reply:
wait did you want to add those as part of the description instead?  I wasn't sure